### ISSUE TITLE
Harbor upgrade failed because the no_proxy setting

### ISSUE THREAD

**Issue by stonezdj** [goharbor/harbor#9944]
When upgrade Harbor Tile 1.8.x to 1.9.3, the upgrading process failed, and login to the failed Harbor VM to start harbor manually,

There are some logs in core.log
```
135 Nov 19 12:05:58 172.80.1.1 core[5878]: 2019-11-19T12:05:58Z [INFO] [/core/main.go:107]: Start to sync quota data .....                                                  
136 Nov 19 12:05:58 172.80.1.1 core[5878]: 2019-11-19T12:05:58Z [INFO] [/core/api/quota/migrator.go:90]: [Quota-Sync]:: start to ping server ... [chart]                    
137 Nov 19 12:06:16 172.80.1.1 core[5878]: 2019-11-19T12:06:16Z [INFO] [/core/api/quota/migrator.go:92]: [Quota-Sync]:: fail to ping server ... [chart], quit sync ...      
138 Nov 19 12:06:16 172.80.1.1 core[5878]: 2019-11-19T12:06:16Z [ERROR] [/core/main.go:109]: Fail to sync quota data, received unexpected status code: 403 <HTML><HEAD>

```

and can not login to Harbor

Check the no_proxy settings in Harbor Tile configure panel:
```
127.0.0.1,localhost,core,registry
```
Workaround:
Change no_proxy to 
```
127.0.0.1,localhost,.local,.internal,log,db,redis,nginx,core,portal,postgresql,jobservice,registry,registryctl,clair
```

**Comment by stonezdj** [comment]
This issue can be found in OSS Harbor because users might upgrade the harbor with harbor.yml, the internal component should be always append to the no_proxy env when rendering this env.

**Comment by ninjadq** [comment]
fixed in #10026 and #9992
